os/board/rtl8721csm/src/component/os/tizenrt/rtk_blemgr.c

```diff
@@ -0,0 +1,419 @@
+
+trble_result_e trble_netmgr_attr_get_data(struct bledev *dev, trble_attr_handle attr_handle, trble_data *data)
+{
+	trble_result_e ret = TRBLE_SUCCESS;
+	data->data = rtw_ble_server_att_get_data_ptr(attr_handle);
+	data->length = rtw_ble_server_att_get_length(attr_handle);
+
+	if (data->data == NULL || data->length == 0) {
+		ret = TRBLE_FAIL;
+	}
+	return ret;
+}
+
+trble_result_e trble_netmgr_attr_reject(struct bledev *dev, trble_attr_handle attr_handle, uint8_t app_errorcode)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_server_reject(attr_handle, app_errorcode);
+	return ret;
+}
+
+trble_result_e trble_netmgr_get_mac_addr_by_conn_handle(struct bledev *dev, trble_conn_handle con_handle, uint8_t bd_addr[TRBLE_BD_ADDR_MAX_LEN])
+{
+	trble_result_e ret = TRBLE_SUCCESS;
+	uint8_t *mac;
+	
+	mac = rtw_ble_server_get_mac_address_by_conn_handle(con_handle);
+	if (mac == NULL) {
+		return TRBLE_FAIL;
+	}
+	memcpy(bd_addr, mac, TRBLE_BD_ADDR_MAX_LEN);
+
+	return ret;
+}
+
+trble_result_e trble_netmgr_get_conn_handle_by_addr(struct bledev *dev, uint8_t bd_addr[TRBLE_BD_ADDR_MAX_LEN], trble_conn_handle *con_handle)
+{
+	trble_result_e ret = TRBLE_SUCCESS;
+	*con_handle = rtw_ble_server_get_conn_handle_by_address(bd_addr);
+	return ret;
+}
+
+trble_result_e trble_netmgr_set_adv_data(struct bledev *dev, trble_data *data)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_server_set_adv_data(data->data, data->length);
+	return ret;
+}
+
+trble_result_e trble_netmgr_set_adv_resp(struct bledev *dev, trble_data *data)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_server_set_adv_name(data->data, data->length);
+	return ret;
+}
+
+trble_result_e trble_netmgr_conn_is_active(struct bledev *dev, trble_conn_handle con_handle, bool *is_active)
+{
+	*is_active = rtw_ble_server_conn_is_active(con_handle);
+	return TRBLE_SUCCESS;
+}
+
+trble_result_e trble_netmgr_conn_is_any_active(struct bledev *dev, bool *is_active)
+{
+	*is_active = rtw_ble_server_conn_is_any_active();
+	return TRBLE_SUCCESS;
+}
+
+trble_result_e trble_netmgr_get_bonded_device(struct bledev *dev, trble_bonded_device_list_s *device_list, uint16_t *device_count)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_server_get_bonded_device(device_list, device_count);
+	return ret;
+}
+
+trble_result_e trble_netmgr_start_adv(struct bledev *dev)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_server_start_adv();
+	return ret;
+}
+
+trble_result_e trble_netmgr_start_adv_directed(struct bledev *dev, uint8_t bd_addr[TRBLE_BD_ADDR_MAX_LEN])
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_server_start_adv_directed(bd_addr);
+	return ret;
+}
+
+trble_result_e trble_netmgr_stop_adv(struct bledev *dev)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_server_stop_adv();
+	return ret;
+}
+
+trble_result_e trble_netmgr_set_adv_interval(struct bledev *dev, uint16_t interval)
+{
+	trble_result_e ret = TRBLE_FAIL;
+	ret = rtw_ble_server_set_adv_interval(interval);
+	return ret;

```

Comment:
is trble_result_e and return of rtw_ble_server compatible?

Reply:
@pillip8282 
Yes, all of rtw_ble functions are using 'trble_' struct & enum.
(https://github.com/Samsung/TizenRT/pull/5021)